packages/snaps-controllers/src/services/webview/WebViewExecutionService.ts

```diff
@@ -1,32 +1,35 @@
-import type { ExecutionServiceArgs } from '../AbstractExecutionService';
-import { ProxyExecutionService } from '../proxy/ProxyExecutionService';
+import type { TerminateJobArgs } from '../AbstractExecutionService';
+import {
+  AbstractExecutionService,
+  type ExecutionServiceArgs,
+} from '../AbstractExecutionService';
 import type { WebViewInterface } from './WebViewMessageStream';
 import { WebViewMessageStream } from './WebViewMessageStream';
 
 export type WebViewExecutionServiceArgs = ExecutionServiceArgs & {
-  getWebView: () => Promise<WebViewInterface>;
+  createWebView: (jobId: string) => Promise<WebViewInterface>;
+  removeWebView: (jobId: string) => void;
 };
 
-export class WebViewExecutionService extends ProxyExecutionService {
-  #getWebView;
+export class WebViewExecutionService extends AbstractExecutionService<string> {
+  #createWebView;
+
+  #removeWebView;
```

Comment:
This will cause a lint violation in ESLint 9.
```suggestion
  readonly #createWebView;

  readonly #removeWebView;
```